pkg/k8shandler/fluentd.go

```diff
@@ -337,6 +354,41 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateFluentdSecret() error
 	return nil
 }
 
+func newLogMetricExporterContainer(cluster *logging.ClusterLogging) v1.Container {
+	collectionSpec := logging.CollectionSpec{}
```

Comment:
remove me.  this is unused and does not make sense in context of the container for the exporter